en-us/api/Microsoft_Toolkit_Uwp_UI_ListViewBaseExtensions.md

```diff
@@ -0,0 +1,28 @@
+---
+permalink: /en-US/api/Microsoft_Toolkit_Uwp_UI_ListViewBaseExtensions.htm
```

Comment:
Never mind, I can see your other PR in the toolkit. Try and keep things a bit separate, it helps to establish what's going where.